shared/tests/utility_module/transaction_test.go

```diff
@@ -52,19 +53,19 @@ func TestUtilityContext_ApplyTransaction(t *testing.T) {
 
 // TODO: Fix this test once txIndexer is implemented by postgres context
 func TestUtilityContext_CheckTransaction(t *testing.T) {
-	// ctx := NewTestingUtilityContext(t, 0)
-	// tx, _, _, _ := newTestingTransaction(t, ctx)
-	// txBz, err := tx.Bytes()
-	// require.NoError(t, err)
-	// require.NoError(t, ctx.CheckTransaction(txBz))
-	// hash, err := tx.Hash()
-	// require.NoError(t, err)
-	// require.True(t, ctx.Mempool.Contains(hash), fmt.Sprintf("the transaction was unable to be checked"))
-	// er := ctx.CheckTransaction(txBz)
-	// require.True(t, er.Error() == types.ErrDuplicateTransaction().Error(), fmt.Sprintf("unexpected err, expected %v got %v", types.ErrDuplicateTransaction().Error(), er.Error()))
+	ctx := NewTestingUtilityContext(t, 0)
```

Comment:
Are these changes intentional in this PR?

Reply:
Genuine mistake, I was trying to understand the flow and I must have used this test during my discovery.

Reverting the change